packages/flutter_tools/templates/create/.analysis_options.tmpl

```diff
@@ -0,0 +1,72 @@
+# Specify analysis options.
+
+# This file is the .analysis_options file used by Flutter editors, such as
+# Atom. It is very similar to flutter_tools/flutter_analysis_options; the only
+# difference (currently) is the public_member_api_docs option, which triggers
+# too many messages to be used in editors.
+
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+  strong-mode: true
```

Comment:
I think we want to turn this on by default. If not now, it will be turned on by default very soon. I'd like to get feedback from users if they run into issues, and the workaround is to manually flip this to false. +1 for defaulting to true.